common/src/main/java/tachyon/worker/DataServerMessage.java

```diff
@@ -356,7 +343,7 @@ public void send(SocketChannel socketChannel) throws IOException {
    *
    * @param lockId The id of the block's locker
    */
-  void setLockId(int lockId) {
+  public void setLockId(int lockId) {
```

Comment:
Right, this is required because I moved DataServerMessage from package tachyon.worker.nio to tachyon.worker in this PR.